src/shared/lib/OncoKbUtils.ts

```diff
@@ -19,8 +20,9 @@ const ONCOGENIC_CLASS_NAMES:{[oncogenic:string]: string} = {
 // (used for sorting purposes)
 const ONCOGENIC_SCORE:{[oncogenic:string]: number} = {
```

Comment:
@alisman Actually, this looks a bit trickier than I initially thought. `oncogenic` is a variable string value coming from data. Trying to get the numerical value for a variable string (`OncogenicScore[oncogenic]`) does not work.